libs/blocks/aside/aside.css

```diff
@@ -29,19 +35,29 @@
 }
 
 .aside.dark {
+  color: white;
   background-color: #1E1E1E;
 }
 
+.aside.light,
+.aside.split {
+  background-color: #f5f5f5;
+}
+
 .aside.dark h1,
 .aside.dark h2,
 .aside.dark h3,
 .aside.dark h4,
 .aside.dark h5,
 .aside.dark h6,
-.aside.dark p {
+.aside.dark p:not(.action-area) {
   color: white;
```

Comment:
Hmm, Id check if this is necessary. Same as above.

Reply:
werent needed!